### ISSUE TITLE
Place order issue after upgraded

### ISSUE THREAD

**Issue by sheldon1310** [JKorf/CoinEx.Net#16]
After upgrade, I always get "Error deserializing data" when placing order. It seems there is deserializing issue for result parsing.

Run time environment:
```
CoinEx.Net version = 3.0.11
CryptoExchange.Net version = 3.0.14
.NetStandard version = 2.0 (It works)
.Net Framework version = 4.6.1 (Error deserializing data)
```

**Comment by JKorf** [comment]
After upgrade of what? CoinEx.Net? Can you check what output is in the logging when this happens?

**Comment by sheldon1310** [comment]
Sorry, my mistake. After I downgrade to previous version, this parsing error still happened. It seems CoinEx API changed caused this parsing issue.

**Comment by muteddisk** [comment]
Are you getting this error no matter what or you getting the message doing something specific?

**Comment by sheldon1310** [comment]
I find strange that the same code causes different result in different platform.

Here is my code in .Net Standard 2.0 environment (it works):

```
        private async static Task Test1()
        {
            CoinExClient client = new CoinExClient();
            client.SetApiCredentials("******", "******");
            var res = await client.PlaceLimitOrderAsync("ALGOUSDT", TransactionType.Buy, 40, 0.3M);
            if (!res.Success)
            {
                Console.WriteLine("Error: " + res.Error.Message);
            }
            else
            {
                Console.WriteLine("Order ID: " + res.Data.Id); // <------ This result printed
            }
        }
```

However, I created a WinForm project (target .Net framework: 4.6.1) and used similar code, it failed to parse response:

```
        private async void btn_test_Click(object sender, EventArgs e)
        {
            CoinExClient client = new CoinExClient();
            client.SetApiCredentials("******", "******");
            var res = await client.PlaceLimitOrderAsync("ALGOUSDT", TransactionType.Buy, 40, 0.3M);
            if (!res.Success)
            {
                MessageBox.Show("Error: " + res.Error.Message); // <------ Got "Error: Error deserializing data"
            }
            else
            {
                MessageBox.Show("Order ID: " + res.Data.Id);
            }
        }
```

I wondering if you could tell me how to open output logging so that I can provide more information for you. Thanks a lot!

**Comment by sheldon1310** [comment]
I got error message as the following:

```
 Deserialize JsonSerializationException: Error setting value to 'FinishTime' on 'CoinEx.Net.Objects.CoinExOrder'.. Received data: {
  "code": 0,
  "data": {
    "id": 29432007135,
    "create_time": 1600506521,
    "finished_time": null,
    "amount": "40.00000000",
    "price": "0.3",
    "deal_amount": "0",
    "deal_money": "0",
    "deal_fee": "0",
    "asset_fee": "0",
    "fee_asset": "USDT",
    "fee_discount": "1.00",
    "avg_price": "0.00",
    "market": "ALGOUSDT",
    "left": "40.00000000",
    "maker_fee_rate": "0.0020",
    "taker_fee_rate": "0.0020",
    "order_type": "limit",
    "type": "buy",
    "status": "not_deal",
    "client_id": ""
  },
  "message": "Success"
}
```

It seems the problem happened at `FinishTime` part.

**Comment by JKorf** [comment]
Should be fixed in version 3.0.12 I just pushed.
